framework/packages/abstract-sdk/src/apis/authz.rs

```diff
@@ -460,13 +469,26 @@ impl AuthZ {
 mod tests {
     use super::*;
 
+    use crate::std::version_control::Account;
     use crate::{apis::stargate::convert_stamp, mock_module::*};
+    use abstract_testing::abstract_mock_querier_builder;
+    use abstract_testing::prelude::*;
     use cosmwasm_std::testing::mock_dependencies;
 
+    pub fn setup() -> (MockDeps, Account, MockModule) {
```

Comment:
This setup fn seems to be the same everywhere and the package already has a `test-utils` feature so IMO it makes sense to pull this fn out of every file into a single location.